src/files-regular/cat-pull-stream.js

```diff
@@ -47,14 +42,8 @@ module.exports = (common, options) => {
         length
       })
 
-      pull(
-        stream,
-        pull.concat((err, data) => {
-          expect(err).to.not.exist()
-          expect(data.toString()).to.equal('lz ')
-          done()
-        })
-      )
+      const data = Buffer.concat(await pullToPromise.any(stream))
```

Comment:
IMO pull-streams pipelines should not be transformed in promises, you lose flexibility and gain 0 value. Pull-streams should either disappear in the future or be left to work as they were created.